src/code/z_parameter.c

```diff
@@ -167,23 +170,21 @@ s16 sExtraItemBases[] = {
     ITEM_NUT,     // ITEM_NUT_UPGRADE_30
 };
 
-s16 D_801BF8DC = 0;
-s16 D_801BF8E0 = 0;
-s16 D_801BF8E4 = 0;
-
-OSTime D_801BF8E8 = 0;
-OSTime D_801BF8F0 = 0;
+s16 sEnvTimerType = PLAYER_ENV_TIMER_NONE;
+s16 sEnvTimerActive = false;
+s16 sPostmanBunnyHoodState = POSTMAN_MINIGAME_BUNNY_HOOD_OFF;
+OSTime sTimerPausedOsTime = 0;
+OSTime sBottleTimerPausedOsTime = 0;
 OSTime D_801BF8F8[] = {
     0, 0, 0, 0, 0, 0, 0,
 };
 OSTime D_801BF930[] = {
     0, 0, 0, 0, 0, 0, 0,
 };
+u8 sIsTimerPaused = false;
+u8 sIsBottleTimerPaused = false;
+s16 sTimerId = 99;
```

Comment:
Is this a magic number?

Reply:
Yeah, it's already part of the enum too. I just missed this one